marklogic-data-hub-central/ui/src/components/entities/mapping/create-edit-mapping-dialog/create-edit-mapping-dialog.test.tsx

```diff
@@ -22,21 +22,21 @@ describe('Create/Edit Mapping Step artifact component', () => {
     expect(getByLabelText('Collection')).toBeInTheDocument();
     expect(getByLabelText('Query')).toBeInTheDocument();
     expect(getByLabelText('collection-input')).toBeInTheDocument();
-    expect(getByText('Save')).toBeDisabled();
+    expect(getByText('Save')).toBeEnabled();
     expect(getByText('Cancel')).toBeEnabled();
     //Collection radio button should be selected by default
     expect(getByLabelText('Collection')).toBeChecked();
   });
 
-  test('Verify mapping name, source query is mandatory and Save button is disabled', async () => {
+  test('Verify mapping name, source query is mandatory and Save button is always enabled', async () => {
     const { getByText, getByPlaceholderText, getByLabelText } = render(<CreateEditMappingDialog {...data.newMap} />);
     const nameInput = getByPlaceholderText('Enter name');
     const saveButton = getByText('Save');
 
     // Enter the value for name input.
     fireEvent.change(nameInput, { target: {value: 'testCreateMap'}});
     expect(nameInput).toHaveValue('testCreateMap');
-    expect(saveButton).toBeDisabled(); // It should be disabled since no value is provided yet for Collection or Query.
+    expect(saveButton).toBeEnabled(); // Consistency change: button is now always enabled; will produce warning instead
 
     //Providing the value for Collection field now so that Save button can be enabled.
```

Comment:
Remove this comment since the Save button now is always enabled. Similarly, remove comments on lines: 49, 70, 85, 89

Reply:
removed all extra comments